### DISCUSSION TITLE
4 Door module / ESP32 version

### DISCUSSION THREAD

**Discussion by CountParadox** [omersiar/esp-rfid#595] [Ideas]
Hey esprfid broski

I have a feature request for an ESP32 version (to use as BLE repeater) and a 4 door version! 

What do you think??

I would love to use this to replace the old Kisi system at my office.

**Comment by matjack1** [comment]
Hey @CountParadox the ESP32 will need a full rewrite, unfortunately.

I'm thinking about doing a prototype on ESP32, but it must be financially viable. Out of curiosity, how much are you spending for the Kisi setup with 4 doors? :)

**Reply by CountParadox** [reply]
I don't know the pricing actually, I'm a tech but we were a reseller so didn't pay, we've dropped it since finally lol
